task13.js

```diff
@@ -0,0 +1,25 @@
+// Write a function that compares two arrays for equality (same elements in the same order).
+
+function isArraysEqual(arr1, arr2) {
+  //   ES6
+  if (arr1.length !== arr2.length) {
```

Comment:
condition is ok, but i would recoment to check if array exist.
solution for `ES6 `-` if (!arr1 || !arr2 || arr1.length !== arr2.length)`
in `ES11 `we can do it as ` if (arr1?.length !== arr2?.length)`